### PR TITLE
Create seed file with survey-sourced responses

### PR DESCRIPTION
**Pull request by BB8bear**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -16,22 +16,150 @@
...
+	stress: [
+		"Stretch.", 
+		"Take a break to play a short game.", 
+		"Go for a walk.", 
+		"Doodle.", 
+		"Write down your lingering thoughts.", 
+		"Focus on structured breathing.", 
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.",
```
Comment (kvrag) [inline db/seeds.rb]: I would change line 51 to "Jog around the block, or longer if you're up for it."

File: db/seeds.rb [hunk 2]
```diff
@@ -16,22 +16,150 @@
...
+		"Go for a walk.", 
+		"Doodle.", 
+		"Write down your lingering thoughts.", 
+		"Focus on structured breathing.", 
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 54

File: db/seeds.rb [hunk 3]
```diff
@@ -16,22 +16,150 @@
...
+		"Focus on structured breathing.", 
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.", 
+		"Take a short nap.", 
+		"Go for a bike ride.", 
+		"Medidate.",
```
Comment (kvrag) [inline db/seeds.rb]: Line 57 - misspelled "meditate"

File: db/seeds.rb [hunk 4]
```diff
@@ -16,22 +16,150 @@
...
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.", 
+		"Take a short nap.", 
+		"Go for a bike ride.", 
+		"Medidate.", 
+		"Go for a walk.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 58 (repeat)

File: db/seeds.rb [hunk 5]
```diff
@@ -16,22 +16,150 @@
...
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.", 
+		"Take a short nap.", 
+		"Go for a bike ride.", 
+		"Medidate.", 
+		"Go for a walk.", 
+		"Have a conversation with friends or family.", 
+		"Go outside.",
```
Comment (kvrag) [inline db/seeds.rb]: Change line 60 to "Go outside or open a window for some fresh air."

File: db/seeds.rb [hunk 6]
```diff
@@ -16,22 +16,150 @@
...
+		"Have a conversation with friends or family.", 
+		"Go outside.", 
+		"Have a glass of water.", 
+		"Play with your pet, or visit an animal shelter.", 
+		"Picture a relaxing scene.", 
+		"Sit in the sun for 15 minutes."
+	],
+	anger: [
+		"Listen to your favorite song.", 
+		"Take a moment to be present in the moment.",
```
Comment (kvrag) [inline db/seeds.rb]: Change line 68 to "Take a moment to center yourself with deep breaths. Clear your mind."

File: db/seeds.rb [hunk 7]
```diff
@@ -16,22 +16,150 @@
...
+		"Have a glass of water.", 
+		"Play with your pet, or visit an animal shelter.", 
+		"Picture a relaxing scene.", 
+		"Sit in the sun for 15 minutes."
+	],
+	anger: [
+		"Listen to your favorite song.", 
+		"Take a moment to be present in the moment.", 
+		"Write a letter you do not intend to send.", 
+		"Take a time out.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 70 - kinda vague

Add -   "Do 10 pushups. Do another 10 if it feels good.",
  "Yell into a pillow."

File: db/seeds.rb [hunk 8]
```diff
@@ -16,22 +16,150 @@
...
+		"Talk with friends or family.", 
+		"Get some fresh air.", 
+		"Go to the gym.", 
+		"Go to the movies."
+	],
+	fear: [
+		"Text a friend.", 
+		"Focus on the present moment.", 
+		"Focus on your breathing.", 
+		"Seek out support.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 84 - same as 81 ish

File: db/seeds.rb [hunk 9]
```diff
@@ -16,22 +16,150 @@
...
+	],
+	fear: [
+		"Text a friend.", 
+		"Focus on the present moment.", 
+		"Focus on your breathing.", 
+		"Seek out support.", 
+		"Go for a walk.", 
+		"Watch a comforting movie.", 
+		"Meditate.", 
+		"Exercise.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 88 - we have lots of exercise ones

File: db/seeds.rb [hunk 10]
```diff
@@ -16,22 +16,150 @@
...
+		"Text a friend.", 
+		"Focus on the present moment.", 
+		"Focus on your breathing.", 
+		"Seek out support.", 
+		"Go for a walk.", 
+		"Watch a comforting movie.", 
+		"Meditate.", 
+		"Exercise.", 
+		"Talk to someone you trust.", 
+		"Change your surroundings.",
```
Comment (kvrag) [inline db/seeds.rb]: Change line 90 to "Change your surroundings, if you can."

File: db/seeds.rb [hunk 11]
```diff
@@ -16,22 +16,150 @@
...
+	joy: [
+		"Go for a walk.", 
+		"Call a friend.", 
+		"Make someone laugh.", 
+		"Hug a loved one.", 
+		"Exercise.", 
+		"Smile at a stranger.", 
+		"Remain in the present.", 
+		"Go outside.",
+		"Mow your lawn."
```
Comment (kvrag) [inline db/seeds.rb]: Delete lines 96, 100, 103, 104 -- add   "Take a moment to feel gratitude for all that you have!",
  "Journal about this moment.",
  "Treat yourself with something you've wanted for a while."

File: db/seeds.rb [hunk 12]
```diff
@@ -16,22 +16,150 @@
...
+		"Write down your thoughts.", 
+		"Take time to get out of the house.", 
+		"Breathe.", 
+		"Spend some time outside.", 
+		"Take a short nap.", 
+		"Drink a cup of tea.", 
+		"Watch a funny movie.", 
+		"Talk to family or friends.", 
+		"Focus your enery on work or a hobby.", 
+		"Spend some time to take care of an easy task."
```
Comment (kvrag) [inline db/seeds.rb]: Edit line 119 - "Take care of an easy task or small chore, like emptying the dishwasher."

Add -   "Cry, if you want to.",
  "Call or text someone you're close to.",
  "Have you eaten recently? Have a healthy snack.",
  "Take a shower or soothing bath."

File: db/seeds.rb [hunk 13]
```diff
@@ -16,22 +16,150 @@
...
+		"Seek out support.", 
+		"Write down your thoughts.", 
+		"Take time to get out of the house.", 
+		"Breathe.", 
+		"Spend some time outside.", 
+		"Take a short nap.", 
+		"Drink a cup of tea.", 
+		"Watch a funny movie.", 
+		"Talk to family or friends.", 
+		"Focus your enery on work or a hobby.",
```
Comment (kvrag) [inline db/seeds.rb]: Edit line 118 - misspelled "energy"

File: db/seeds.rb [hunk 14]
```diff
@@ -16,22 +16,150 @@
...
+		"Speak with someone you trust.", 
+		"Seek out a friend or loved one.", 
+		"Go for a walk.", 
+		"Treat yourself to a snack that you enjoy.", 
+		"Meditate.", 
+		"Take a deep breath.", 
+		"Stretch.", 
+		"Go for a run.", 
+		"Listen to relaxing music.", 
+		"Make a list."
```
Comment (kvrag) [inline db/seeds.rb]: Delete lines 122, 123, 130, 131

Add - 
"Inhale as deeply as you can, and exhale slowly.",
  "Drink a cold glass of water or milk.",
  "Interact with your pet, or visit an animal shelter.",
  "Take a moment to stretch.",
  "Take stock of your immediate environment, and change one thing for the better."

File: db/seeds.rb [hunk 15]
```diff
@@ -16,22 +16,150 @@
...
+	],
+	disgust: [
+		"Spend a moment to consider the root of your current emotional state.", 
+		"Go for a walk.", 
+		"Try to use your empathy.", 
+		"Express your feelings in a constructive way.", 
+		"Remove yourself from your current surroundings.",
+		"Have some ice cream."
```
Comment (kvrag) [inline db/seeds.rb]: Delete line 142

File: db/seeds.rb [hunk 16]
```diff
@@ -16,22 +16,150 @@
...
+	],
+	tired: [
+		"Get up and move around.", 
+		"Drink water.",
```
Comment (kvrag) [inline db/seeds.rb]: Edit line 146 - "Drink a cold glass of water."

File: db/seeds.rb [hunk 17]
```diff
@@ -16,22 +16,150 @@
...
+		"Remove yourself from your current surroundings.",
+		"Have some ice cream."
+	],
+	tired: [
+		"Get up and move around.", 
+		"Drink water.", 
+		"Stay active.", 
+		"Go for a walk.", 
+		"Get a cup of coffe.", 
+		"Have a cup of tea.",
```
Comment (kvrag) [inline db/seeds.rb]: Delete lines 147 and 150, edit line 149 - "Have a cup of coffee or your favorite tea."

File: db/seeds.rb [hunk 18]
```diff
@@ -16,22 +16,150 @@
...
+		"Stay active.", 
+		"Go for a walk.", 
+		"Get a cup of coffe.", 
+		"Have a cup of tea.", 
+		"Tell a joke.", 
+		"Take a short nap.", 
+		"Meditate.", 
+		"Exercise.", 
+		"Go for a run.", 
+		"Take a break for a few moments."
```
Comment (kvrag) [inline db/seeds.rb]: Delete lines 152, 153, 154, 155

Add - 
  "Did you get much sleep? Try closing your eyes for 20 minutes.",
  "Take a shower, or wash your face in cold water."

File: db/seeds.rb [hunk 19]
```diff
@@ -16,22 +16,150 @@
...
+	]
+}
+
+actions.each do |emote, action_array|
+	action_array.each do |action|
+		Action.create(content: "#{action} (#{emote})")
```
Comment (kvrag) [inline db/seeds.rb]: Delete `(#{emote})")` - we don't want to display the top emotion

**Review by kvrag — DISMISSED** [review]

**Review by caristopmer — APPROVED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (BB8bear) [reply db/seeds.rb]: Updates have been made. Please review new pull request.

File: db/seeds.rb (hunk 2 above)
Comment (BB8bear) [reply db/seeds.rb]: Updates have been made. Please review new pull request.

File: db/seeds.rb (hunk 3 above)
Comment (BB8bear) [reply db/seeds.rb]: Updates have been made. Please review new pull request.

File: db/seeds.rb [hunk 20]
```diff
@@ -16,22 +16,143 @@
...
+		"Have a conversation with friends or family.", 
+		"Go outside or open a window for fresh air.", 
+		"Have a glass of water.", 
+		"Play with your pet, or visit an animal shelter.", 
+		"Picture a relaxing scene.", 
+		"Sit in the sun for 15 minutes."
+	],
+	anger: [
+		"Listen to your favorite song.", 
+		"Take a moment to be center yourself with deep breaths.Clear your mind.",
```
Comment (kvrag) [inline db/seeds.rb]: Edit to: "Take a moment to center yourself with deep breaths. Clear your mind.",

File: db/seeds.rb [hunk 21]
```diff
@@ -16,22 +16,143 @@
...
+		"Make someone laugh.", 
+		"Hug a loved one.", 
+		"Smile at a stranger.", 
+		"Remain in the present.",
+		"Take a moment to feel gratitude for all that you have!",
+		"Journal about this moment.", 
+		"Treat yourself with something you've wanted for a while."
+	],
+	sadness: [
+		"Do some push-ups.",
```
Comment (kvrag) [inline db/seeds.rb]: delete line 101 and 103

**Review by kvrag — CHANGES_REQUESTED** [review]
Just a coupla small things!

File: db/seeds.rb (hunk 20 above)
Comment (BB8bear) [reply db/seeds.rb]: Fixed

File: db/seeds.rb (hunk 21 above)
Comment (BB8bear) [reply db/seeds.rb]: Fixed

**Review by kvrag — APPROVED** [review]
Coo!!1

**Pull request merged**